bokeh/sphinxext/_templates/bokehjs_html_template.html

```diff
@@ -5,8 +5,11 @@
 <title>Complete Example</title>
 {% for css_file in css_files %}
 <link rel="stylesheet" href="{{css_file}}" type="text/css" />{% endfor %}
-{% for js_file in js_files %}
-<script type="text/javascript" src="{{js_file}}"></script>{% endfor %}
+{%- for js_file in js_files %}
+<script type="text/javascript" src="{{ js_file }}"
```

Comment:
What’s the rule on spacing in the curly braces? Can change this back to `{{js_file}}`.

Reply:
I don't know what that we have any specific preference expressed, I'm personally fine with adding the spaces